src/display/text_layer.js

```diff
@@ -63,6 +63,7 @@ var renderTextLayer = (function renderTextLayerClosure() {
     // Initialize all used properties to keep the caches monomorphic.
     var textDiv = document.createElement('div');
     var textDivProperties = {
+      style: '',
```

Comment:
I meant putting `if (divProperties.originalTransform) { ... }` around that snippet. Transforming with an empty string doesn't make sense to me, so it would be more efficient if we would just skip that.